ui/webui/test/check-storage

```diff
@@ -377,6 +377,17 @@ class TestUtils():
 
         return True
 
+    def check_mountpoint_table_row(self, b, row, mountpoint=None, device=None, reformat=None, format_type=None):
```

Comment:
You should be able to drop `b` and use just `self.browser`, I think.

The file is too large to read in github review, sadly, so I apologize if that's actually not the case.